Rider/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,4 @@
-#Tue Oct 22 17:31:47 EDT 2019
+#Sun Oct 27 00:36:55 EDT 2019
```

Comment:
Perhaps this file should also not be committed

Reply:
Gradle wrapper is designed to be committed. We should probably avoid unnecessary changes here.